src/activities/ActivityUsageEntity.js

```diff
@@ -264,4 +264,126 @@ export class ActivityUsageEntity extends Entity {
 	isPublished() {
 		return this._entity && this._entity.hasClass(Classes.activities.draftPublishedEntity) && this._entity.hasClass(Classes.activities.published);
 	}
+
+	/**
+	 * @returns {string} Score out of value of the activity usage
+	 */
+	scoreOutOf() {
+		if (this.canEditScoreOutOf()) {
+			const action = this._getScoreOutOfAction();
+			if (action.hasFieldByName('scoreOutOf')) {
+				return action.getFieldByName('scoreOutOf').value;
+			}
+		}
+		return this._getScoreOutOfEntity().properties.scoreOutOf;
```

Comment:
Kind of an edge case concern, but `_getScoreOutOfEntity` can return `null` (if `getSubEntity` doesn't return anything), which would cause `.properties` to blow up.

Reply:
Right, will add a check there...